modules/olm-adding-pullsecret-catalogsource.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * operators/admin/olm-restricted-networks.adoc
+// * operators/admin/managing-custom-catalogs.adoc
+
+[id="olm-adding-pullsecret-catalogsource_{context}"]
+= Adding pull secrets to catalog sources
+
+If an index image or catalog image is hosted on an authenticated container image registry, also known as a private registry, you can create a pull secret that contains the authentication credentials for the registry. By referencing the secret in a catalog source, the required images can be pulled for use in OperatorHub.
+
+.Prerequisites
+
+* An index image or catalog image built and pushed to a private registry.
+
+.Procedure
+
+. Create a secret that contains the authentication credentials for the private registry:
++
+[source,terminal]
+----
+$ oc create secret generic <secret_name> \
+    --from-file=.dockerconfigjson=/root/.docker/config.json \
```

Comment:
If using this `/root/.docker/config.json` file, we'd better add a login step to get the private auth info, such as:
```console
$ docker login <registry>:<port>
```
Or we can replace this  `/root/.docker/config.json` file with other auth files.